Versie2020 05 (#8)
inbo/n2khab-droogtemeetnet: 02_Methodiek.Rmd

Deze rasters worden gebruikt in de meetnetten natuurlijk milieu en in het meetnet biotische habitatkwaliteit.
    - Voor de ruimtelijke spreiding van meetpunten in het meetnet kiezen we voor rastercellen met resolutie 8 km (8192 m, level 8).
In totaal zijn er 262 cellen in Vlaanderen, waarvan 149 volledig in Vlaanderen liggen.
    - Het gewenste totaal aantal meetpunten = `r tot_n_tub `  

> - We willen daarenboven ook gestratificeerd werken volgens de 5-delige GT-groepsindeling, om zo te verzekeren dat elke GT-groep voldoende gemonitord zal worden.

Reviewer: ```suggestion
- We willen daarenboven ook gestratificeerd werken volgens de GT-groepsindeling, om zo te verzekeren dat elke GT-groep voldoende gemonitord zal worden.
```

We geven niet alle GT-groepen een even groot gewicht. Vegetatietypen van GT-klasse 5 worden niet meegenomen. 

> Van deze GT-klasse is immers de grondwaterafhankelijkheid plaatsgebonden en (soms heel) variabel.

Reviewer: ```suggestion
Van deze GT-klasse is immers de grondwaterafhankelijkheid plaatsgebonden en (soms heel) variabel (zie hoofdstuk \@ref(inleiding)).
```

De twee natste GT-klassen krijgen een lager gewicht toegemeten, omdat deze klassen vermoedelijk het traagst een droogte-signaal zullen geven. 

Concreet:

> - Het aantal stratificatielagen  = `r aantal_strat `

Reviewer: ```suggestion
- Het aantal strata  = `r aantal_strat `
```

Reviewer: This turns out to be '5', is that still intended?

Author: Theoretically it's 5, but in practice 4. I changed it to 4

- De verdeling van het aantal meetpunten per laag is gegeven in tabel \@ref(tab:GTgroepen-verdeling-over-strata): 
```{r GTgroepen-verdeling-over-strata}
kobj <- kable(gw_types_groups %>% arrange(`GT-groep: nummer`), 
      caption = "Verdeling van het gewenst aantal meetlocaties over de grondwatertype-groepen") 

if (pdfmaken == TRUE) {
  kobj
} else {
  kobj   %>% 
    kable_styling(bootstrap_options = c("striped", "hover", "condensed", "responsive"), 
                  full_width = TRUE, 
                  position = "left",

>                   font_size = 9,

Reviewer: A personal note: I find the font in html tables quite small. In my experience, just using a default `kable_styling()` yields an acceptable visual result in `html_document2`.

For more narrow tables (as this one) I'd use `full_width = FALSE` or `full_width = FALSE, position = "left"`. But it's also a matter of taste (and we actually need conventions, maybe they're in `INBOmd` already)

For scroll boxes I had good results with

```r
%>% 
kable_styling(font_size = 12) %>% 
    scroll_box(height = "500px")
```

* het verlengen of invullen van te korte of onregelmatige grondwaterstandsreeksen
* het opsporen van lineaire trends

Het eerste doel is ook al hoger ter sprake gekomen: voor het berekenen en evalueren van indicatoren is het nodig te kunnen beschikken over relatief lange meetreeksen. 
We zoeken meetreeksen die aanvangen in 1985 of eerder.
Slagen we er in om op een betrouwbare wijze de reeksen vanaf 1985 te verlengen/vervolledigen, dan kan een meetpunt geschikt worden om in het droogtemeetnet opgenomen te worden. 
De tijdreeksen krijgen ook een uniforme lengte, waardoor het opvolgen van de evolutie van de droogte-indicator over de jaren heen meer betekenis krijgt.

Het tweede doel is in feite een bijkomend selectiecriterium. 

> Indien uit een tijdreeksanalyse blijkt dat, na rekening te houden met zoveel mogelijk factoren (maar in de praktijk enkel met het weer), de niet door het weer verklaarde component van de waterstanden van een meetpunt geleidelijk aan dalen of stijgen, leent dat meetpunt zich niet goed voor het berekenen van een droogte-indicator. 

Reviewer: ```suggestion
Indien uit een tijdreeksanalyse blijkt dat, na rekening te houden met zoveel mogelijk factoren (maar in de praktijk enkel met het weer), de niet door het weer verklaarde component van de waterstanden van een meetpunt geleidelijk aan daalt of stijgt, leent dat meetpunt zich niet goed voor het berekenen van een droogte-indicator. 
```

Het eerste doel is ook al hoger ter sprake gekomen. 
Slagen we er in om op een betrouwbare wijze de waterstandsreeksen te verlengen/vervolledigen, dan kan een meetpunt geschikt worden om in het droogtemeetnet opgenomen te worden. 
De tijdreeksen krijgen ook een uniforme lengte, waardoor het opvolgen van de evolutie van de droogte-indicator over de jaren heen meer betekenis krijgt.

> Het tweede doel is in feite een bijkomende selectiecriterium. Indien uit een tijdreeksanalyse blijkt dat, na het in rekening brengen met zoveel mogelijk factoren (maar in de praktijk enkel met het weer), de waterstanden van een meetpunt geleidelijkaan dalen of stijgen, leent dat meetpunt zich niet goed voor het berekenen van een droogte-indicator. 

Reviewer: ```suggestion
Het tweede doel is in feite een bijkomend selectiecriterium. Indien uit een tijdreeksanalyse blijkt dat, na rekening te houden met zoveel mogelijk factoren (maar in de praktijk enkel met het weer), de waterstanden van een meetpunt geleidelijk aan dalen of stijgen, leent dat meetpunt zich niet goed voor het berekenen van een droogte-indicator. 
```

Reviewer: > na rekening te houden met zoveel mogelijk factoren (maar in de praktijk enkel met het weer), de waterstanden van een meetpunt geleidelijk aan dalen of stijgen

"de waterstanden van een meetpunt": in deze context beter vervangen door "de niet door het weer verklaarde component van de waterstanden van een meetpunt"?

Daarmee maak je ook duidelijker dat de _waterstanden_ an sich _wel_ een trend mogen vertonen, zolang die maar verklaard zijn door een trend in het weer. Je hebt het dus over een partiële trend (na isolatie van andere, eveneens tijdsgebonden effecten). Daarin zit volgens mij de verwarring voor de lezer, misschien kan dat nog explicieter.

Author: Thanks for your help to clarify this.
